### ISSUE TITLE
ELI5 where is $class, $method coming from? I cant see any reference for it, is that an error?

### ISSUE THREAD

**Issue by DonJorgen** [facade/ignition#36]
https://github.com/facade/ignition/blob/7017073cd9a46c6798ee18362e259b5509ec9b80/src/SolutionProviders/BadMethodCallSolutionProvider.php#L38

**Comment by mpociot** [comment]
It is coming from PHPs extract method:

https://github.com/facade/ignition/blob/7017073cd9a46c6798ee18362e259b5509ec9b80/src/SolutionProviders/BadMethodCallSolutionProvider.php#L44

See here for more information https://www.php.net/manual/en/function.extract.php
